SetGame.xcodeproj/project.pbxproj

```diff
@@ -7,13 +7,16 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		194DA4F926EE4DC3002392B5 /* CardsMatching.swift in Sources */ = {isa = PBXBuildFile; fileRef = 194DA4F826EE4DC3002392B5 /* CardsMatching.swift */; };
+		194DA4FB26EE4DD1002392B5 /* CardsMatchingTest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 194DA4FA26EE4DD1002392B5 /* CardsMatchingTest.swift */; };
+		194DA4FD26EE5605002392B5 /* ValuesMatchingTest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 194DA4FC26EE5605002392B5 /* ValuesMatchingTest.swift */; };
 		1978206626AAC30700624E75 /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1978206526AAC30700624E75 /* AppDelegate.swift */; };
 		1978206826AAC30700624E75 /* SceneDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1978206726AAC30700624E75 /* SceneDelegate.swift */; };
 		1978206A26AAC30700624E75 /* ViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1978206926AAC30700624E75 /* ViewController.swift */; };
 		1978206D26AAC30700624E75 /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 1978206B26AAC30700624E75 /* Main.storyboard */; };
 		1978206F26AAC30700624E75 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 1978206E26AAC30700624E75 /* Assets.xcassets */; };
 		1978207226AAC30700624E75 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 1978207026AAC30700624E75 /* LaunchScreen.storyboard */; };
-		1978207D26AAC30700624E75 /* SetGameTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1978207C26AAC30700624E75 /* SetGameTests.swift */; };
+		1978207D26AAC30700624E75 /* SetGameModelTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1978207C26AAC30700624E75 /* SetGameModelTests.swift */; };
```

Comment:
Wouldn't it be a good idea to have all the renamings in a separate commit just to easify the reviews?